### PR TITLE
Update changelog for 1.37.2

### PR DESCRIPTION
**Pull request by nohwnd**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 All changes to the project will be documented in this file.
 
 ## [1.37.2] - Not Yet Released
+* Mono versions update in the readme (PR: [#1929](https://github.com/OmniSharp/omnisharp-roslyn/pull/1929))
```
Comment (filipw) [inline CHANGELOG.md:5]: we only add changes that are visible for consumers as features/bugs and skip things like readme/documentation updates, build script changes or test infra modifications. could you please get rid of them?

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +15,16 @@ All changes to the project will be documented in this file.
...
 * Add support for new completion endpoints when working with Cake ([#1939](https://github.com/OmniSharp/omnisharp-roslyn/issues/1939), PR: [#1944](https://github.com/OmniSharp/omnisharp-roslyn/pull/1944))
+* When an analyzer fails to load, log an error (PR: [#1972](https://github.com/OmniSharp/omnisharp-roslyn/pull/1972))
+* Bump Roslyn to 3.8.0-4.20503.2 (PR: [#1974](https://github.com/OmniSharp/omnisharp-roslyn/pull/1974))
+* Updated build tool version for NET 5 RC2 (PR: [#1971](https://github.com/OmniSharp/omnisharp-roslyn/pull/1971))
+* Added support for 'extract base class' (PR: [#1969](https://github.com/OmniSharp/omnisharp-roslyn/pull/1969))
+* Moved most test asset projects to .NET Core 3.1 (PR: [#1968](https://github.com/OmniSharp/omnisharp-roslyn/pull/1968))
+* Upgraded to SDK 3.1.402 (PR: [#1966](https://github.com/OmniSharp/omnisharp-roslyn/pull/1966))
+* OmniSharp.Path can only be set in user settings (PR: [#1946](https://github.com/OmniSharp/omnisharp-roslyn/pull/1946))
+* Adds support for /completion and /completion/resolve endpoints for Cake. (PR: [#1944](https://github.com/OmniSharp/omnisharp-roslyn/pull/1944))
```
Comment (filipw) [inline CHANGELOG.md:25]: this is  duplicate

File: CHANGELOG.md (hunk 1 above)
Comment (nohwnd) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 2 above)
Comment (nohwnd) [reply CHANGELOG.md]: I didn't see what this was duplicate of but hopefully with the update the problem is solved.

**Review by filipw — APPROVED** [review]
thanks

**Pull request merged**